docs/docs/developers/guides/js_apps/how_to_send_transaction.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Sending Transactions
+sidebar_position: 4
+description: Step-by-step guide to sending transactions to Aztec contracts using Aztec.js.
+tags: [transactions, contracts]
+---
+
+This guide shows you how to send transactions to smart contracts on Aztec.
+
+## Prerequisites
+
+- Deployed contract address and ABI
+- Funded wallet account
+- Running Aztec sandbox or network
+- Understanding of [contract interactions](../smart_contracts/how_to_call_contracts.md)
+
+## Connect to a deployed contract
+
+### Step 1: Import required modules
+
+```typescript
+import { Contract } from '@aztec/aztec.js';
+```
+
+### Step 2: Get contract instance
+
+```typescript
+const contract = await Contract.at(contractAddress, ContractArtifact, wallet);
+```
+
+The contract instance provides methods to interact with all the contract's functions.
+
+## Send a basic transaction
+
+### Step 1: Call a contract method
+
+```typescript
+await contract.methods.my_function(param1, param2)
+    .send({ from: senderAddress })
+    .wait();
+```
+
+### Step 2: Verify transaction success
+
+```typescript
+const receipt = await contract.methods.my_function(param1, param2)
+    .send({ from: senderAddress })
+    .wait();
+
+console.log(`Transaction completed in block ${receipt.blockNumber}`);
+console.log(`Transaction hash: ${receipt.txHash.toString()}`);
+```
+
+## Send transactions with different options
+
+### Specify gas settings
+
+```typescript
+await contract.methods.my_function(param1)
+    .send({ 
+        from: senderAddress,
+        gasSettings: {
+            gasLimit: 1000000n,
+            maxFeesPerGas: {
+                feePerL2Gas: Fr.from(10),
+                feePerDaGas: Fr.from(10),
+            }
+        }
+    })
+    .wait();
+```
+
+### Send transaction without waiting
+
+```typescript
+// Send transaction and get transaction request
+const tx = contract.methods.my_function(param1).send({ from: senderAddress });
+
+// Get transaction hash immediately
+const txHash = await tx.getTxHash();
+console.log(`Transaction sent: ${txHash}`);
+
+// Wait for inclusion later
+const receipt = await tx.wait();
+```
+
+## Handle private transactions
+
+### Step 1: Execute private function
+
+```typescript
+await contract.methods.private_function(secretParam1, secretParam2)
+    .send({ from: senderAddress })
+    .wait();
+```
+
+### Step 2: Execute private-to-public function
+
+```typescript
+// Private function that enqueues public calls
+await contract.methods.private_to_public_function(param1)
+    .send({ from: senderAddress })
+    .wait();
+```
+
+## Handle public transactions
+
+### Execute public function
```

Comment:
Is it worth specifying `contract.methods.private_function`, `contract.methods.private_to_public_function` and `contract.methods.public_function`? they are all the same syntax